docs/source/installation/versions.rst

```diff
@@ -26,21 +26,21 @@ The three versions of Manim
 
 There are currently three main versions of manim. They are as follows:
 
-- **ManimCE**: The community edition of manim. This is the version documented by this website, and is named `manim <https://pypi.org/project/manim/https://pypi.org/project/manim/>`_ on pip.
-- `ManimGL <https://github.com/3b1b/manim>`_: The current version of manim that is used by 3blue1brown. It supports OpenGL rendering and interactivity, and is named ``manimgl`` on pip. You can find documentation for it `here <https://3b1b.github.io/manim/index.html>`_.
+- **ManimCE**: The community edition of manim. This is the version documented by this website and is named `manim <https://pypi.org/project/manim/https://pypi.org/project/manim/>`_ on pip.
+- `ManimGL <https://github.com/3b1b/manim>`_: The current version of manim that is used by 3blue1brown. It supports OpenGL rendering and interactivity and is named ``manimgl`` on pip. You can find documentation for it `here <https://3b1b.github.io/manim/index.html>`_.
 - `ManimCairo <https://github.com/3b1b/manim/tree/cairo-backend>`_: The old version of manim originally used by 3blue1brown. It is not available on pip.
 
 Which version to use
 ********************
 We recommend using the community edition for most purposes, as it has been developed to be more stable, 
 better tested, quicker to respond to community contributions, and easier for beginners to use. 
-It also has partial experimental OpenGL support, and should have full support shortly (as of April, 2021).
+It also has partial experimental OpenGL support and should have full support shortly (as of April 2021).
 
 If you would like to use a version with full OpenGL support or render recent 3Blue1Brown videos (2020 onwards), you should use ManimGL.
 
 If you would like to render old 3Blue1Brown projects (2019 and before), you should use ManimCairo.
 
-Notes on installation, documentation and use
+Notes on installation, documentation, and use
```

Comment:
just hope there isn't a debate on all these usages of the oxford comma...

Reply:
Lol